misc/dist/osx_tools.app/Contents/Info.plist

```diff
@@ -90,6 +90,57 @@
 				<string>scene/x-scn</string>
 			</dict>
 		</dict>
+		<dict>
+			<key>UTTypeIdentifier</key>
+			<string>public.gd</string>
+			<key>UTTypeReferenceURL</key>
+			<string></string>
+			<key>UTTypeDescription</key>
+			<string>Godot Script</string>
+			<key>UTTypeIconFile</key>
+			<string>GDScript.icns</string>
+			<key>UTTypeConformsTo</key>
+			<array>
+				<string>public.data</string>
+			</array>
+			<key>UTTypeTagSpecification</key>
+			<dict>
+				<key>com.apple.ostype</key>
+				<string>GDSC</string>
+				<key>public.filename-extension</key>
+				<array>
+					<string>gd</string>
+				</array>
+				<key>public.mime-type</key>
+				<string>script/x-gd</string>
```

Comment:
Can we be more explicit on the declare MIME type identifiers too? e.g. `script/x-gdscript`.

BTW is `script/` the correct category?
On Linux I have:
```
$ echo "/usr/bin/python" > test.py && mimetype test.py
test.py: text/x-python
```

Reply:
Changed to `text/x-gdscript`, `application/x-godot-resource`, `application/x-godot-scene` and `text/x-godot-project`